containerization/docker/fedora/Dockerfile

```diff
@@ -0,0 +1,41 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2020-2022 Intel Corporation.
+
+FROM fedora:35 AS build
```

Comment:
not sure if we want the multistage build here - but could add the 2nd half similar to the ubuntu build if needed

Reply:
I suspect the size is probably similar between the build and runtime container images since the base image will most likely dominate the overall image size.

For development purposes, and as an example Dockerfile, this is good. Actually I prefer having the single stage for development since I can debug there too; so I'd almost prefer if the Ubuntu dockerfile just had a single stage. Not too big a deal, at least for me, though.